config/cowbird.example.ini

```diff
@@ -26,6 +26,9 @@ pyramid.debug_routematch = false
 pyramid.default_locale_name = en
 pyramid.includes =
 
+## mongodb
+mongo_uri = mongodb://0.0.0.0:27017/cowbird
+
```

Comment:
Those two are not meant to be public. 
`USE_PYRAMID_CELERY_APP_CFG`, is set to false for testing purpose (because we used the mocked celery app), but the pyramid celery ap must be used otherwise. 
`USE_CELERY_CFG` is set to false when cowbird is run as a cli and must be true when run as a service.

I guess I should use names more related to the purpose than the effect, something like `USE_TEST_CELERY_APP` and `CLI_MODE`

Reply:
Sound good.